phpBB/styles/prosilver/template/ucp_register.html

```diff
@@ -38,7 +38,7 @@ <h2>{SITENAME} - {L_REGISTRATION}</h2>
 	<!-- EVENT ucp_register_credentials_before -->
 	<dl>
 		<dt><label for="username">{L_USERNAME}{L_COLON}</label><br /><span>{L_USERNAME_EXPLAIN}</span></dt>
-		<dd><input type="text" tabindex="1" name="username" id="username" size="25" value="{USERNAME}" class="inputbox autowidth" title="{L_USERNAME}" /></dd>
+		<dd><input type="text" tabindex="1" name="username" id="username" size="25" value="{USERNAME}" class="inputbox autowidth" title="{L_USERNAME}" autocomplete="username" /></dd>
```

Comment:
I don't think it makes sense to have autocomplete while registering

Reply:
ok i'll switch it to off